app/i18n/locales/ru.js

```diff
@@ -238,6 +238,9 @@ export default {
 	SAVE: 'СОХРАНИТЬ',
 	Save_Changes: 'Сохранить изменения',
 	Save: 'Сохранить',
+	Save_To_Camera_Error: 'Ошибка',
```

Comment:
Always translate only the languages you speak to avoid misuse or other damages.

Reply:
Ok. Sorry. I need to remove that?

Reply:
Yes. Leave it only pt-BR and en.

Reply:
Hi @diegolmello, I removed translates.